tests/unit/src/aws_iot_tests_unit_shadow_action_helper.c

```diff
@@ -838,9 +838,75 @@ TEST_C(ShadowActionTests, NoClientTokenForShadowAction) {
 	IOT_DEBUG("-->Success - No client token for shadow action \n");
 }
 
+TEST_C(ShadowActionTests, IsJsonValid)
+{
+	bool ret_val;
+	char getRequestJson[TEST_JSON_SIZE];
+	
+	IOT_DEBUG("-->Running Shadow Action Tests - sReceivedJsonValid \n");
```

Comment:
Wrong test name.

Reply:
Fixed

Reply:
This is still `sReceivedJsonValid` in the latest version?